content-src/lib/getVideoPreview.js

```diff
@@ -1,32 +0,0 @@
-const getYouTubeID = require("get-youtube-id");
```

Comment:
If we do want to try and integrate something like `depcheck-ci`, we could use this as our baseline .depcheckrc file:

```json
{
  "ignore": [
    "babel",
    "babel-*",
    "depcheck-ci",
    "eslint",
    "eventemitter2",
    "husky",
    "istanbul-instrumenter-loader",
    "karma-*",
    "mocha",
    "raw-loader",
    "redux-logger",
    "requirejs",
    "sinon",
    "style-loader"
  ]
}
```

With that config, we would have seen the following fail on **get-youtube-id** with an exit code of 1:

```sh
$ ./node_modules/.bin/depcheck-ci

error: You have unused dependencies:

get-youtube-id
```